src/sc_client/session.py

```diff
@@ -161,11 +198,21 @@ def send_message(request_type: common.ClientCommand, payload: Any) -> Response:
             common.PAYLOAD: payload,
         }
     )
-    len_data = len(bytes(data, "utf-8"))
-    if len_data > MAX_PAYLOAD_SIZE:
-        raise PayloadMaxSizeError(f"Data is too large: {len_data} > {MAX_PAYLOAD_SIZE} bytes")
-    _send_message(data)
-    response = receive_message(command_id)
+
+    def handle_request() -> Response:
+        len_data = len(bytes(data, "utf-8"))
+        if len_data > MAX_PAYLOAD_SIZE:
+            _on_error(
+                _ScClientSession.ws_app,
+                PayloadMaxSizeError(f"Data is too large: {len_data} > {MAX_PAYLOAD_SIZE} bytes")
+            )
```

Comment:
This would force an error handler and a reconnect while the websocket itself is still perfectly valid. Please restrain from using callback functions manually and use a separate method for raising errors defined by `py-sc-client`'s logic

Reply:
No, this is a different type of error and does not cause a reconnect.